tests/upgrade_path/test_upgrade_path.py

```diff
@@ -10,10 +10,13 @@
 from datetime import datetime
 
 from tests.common.fixtures.ptfhost_utils import copy_ptftests_directory   # lgtm[py/unused-import]
-from tests.common.fixtures.ptfhost_utils import change_mac_addresses      # lgtm[py/unused-import]
```

Comment:
isn't this needed?

Reply:
This required as all interface on new ptfdocker would have the same mac address.

Reply:
Thanks. I'll recover it back.